docs/porting/display.rst

```diff
@@ -262,7 +262,7 @@ Further reading
    for a template for your own driver.
 -  :ref:`Drawing <drawing>` to learn more about how rendering
    works in LVGL.
--  :ref:`Display features <disp>` to learn more about higher
+-  :ref:`Display features <display_features>` to learn more about higher
```

Comment:
same thing with this one.

:ref:\`display_features\`

Reply:
oh yes, good catch